workloads/hyper-scale/common.sh

```diff
@@ -10,21 +10,23 @@ prep(){
         export PATH=$PATH:/usr/local/go/bin
     fi
     if [[ ${HYPERSHIFT_CLI_INSTALL} != "false" ]]; then
-        echo "Remove current Hypershift CLI directory.."
-        sudo rm -rf hypershift || true
+        echo "Clean-up Hypershift binaries.."
         sudo rm /usr/local/bin/hypershift || true
-        git clone -q --depth=1 --single-branch --branch ${HYPERSHIFT_CLI_VERSION} ${HYPERSHIFT_CLI_FORK}    
-        pushd hypershift
+        export TEMP_DIR=$(mktemp -d)
+
+        git clone -q --depth=1 --single-branch --branch ${HYPERSHIFT_CLI_VERSION} ${HYPERSHIFT_CLI_FORK} -v $TEMP_DIR/hypershift
+        pushd $TEMP_DIR/hypershift
         make build
         sudo cp bin/hypershift /usr/local/bin
```

Comment:
This will still overwrite the current CLI version in use.

Reply:
+1